src/richie/apps/courses/defaults.py

```diff
@@ -6,3 +6,50 @@
 PAGE_EXTENSION_TOOLBAR_ITEM_POSITION = getattr(
     settings, "RICHIE_PAGE_EXTENSION_TOOLBAR_ITEM_POSITION", 4
 )
+
+BLOGPOSTS_PAGE = {
```

Comment:
Can you add a comment to this section explaining the purpose of those defaults?

Reply:
done